packages/tests/tests/multiaddr.node.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { multiaddr } from "@multiformats/multiaddr";
+import type { Waku } from "@waku/interfaces";
+import { createLightNode } from "@waku/sdk";
+import { expect } from "chai";
+
+import { NimGoNode } from "../src/index.js";
+
+describe("dials multiaddr", function () {
+  let waku: Waku;
+  let nwaku: NimGoNode;
+
+  afterEach(async function () {
+    !!nwaku &&
+      nwaku.stop().catch((e) => console.log("Nwaku failed to stop", e));
+    !!waku && waku.stop().catch((e) => console.log("Waku failed to stop", e));
+  });
+
+  it("TLS", async function () {
+    this.timeout(20_000);
+
+    let tlsWorks = true;
+
+    waku = await createLightNode();
+    await waku.start();
+    try {
+      // dummy multiaddr, doesn't have to be valid
+      await waku.dial(multiaddr(`/ip4/127.0.0.1/tcp/30303/tls/ws`));
```

Comment:
do we need to test other `multiaddrs` here such as `p2p-circuit` or the special case of it that @richard-ramos was mentioning [here](https://discord.com/channels/1110799176264056863/1147289748109541396/1148234990098853990)?

Reply:
cool, added not to forget there https://github.com/waku-org/js-waku/pull/1531#issuecomment-1731165912